src/PaymentGateways/Gateways/Stripe/BECSGateway.php

```diff
@@ -117,4 +122,12 @@ protected function getPaymentIntentArgs()
             ],
         ];
     }
+
+    /**
+     * @unreleased
+     * @inerhitDoc
+     */
+    public function refundDonation(Donation $donation)
+    {
+    }
```

Comment:
@ravinderk I understand you put these here because if you don't it will throw an error since the interface requires it. Would you please put a temporary exception in them? Something like:
```php
throw new Exception('Method has not been implemented yet. Please use the legacy method in the meantime.');
```

I just don't want someone to accidentally use one of these before they're ready.

Reply:
Exception added: https://github.com/impress-org/givewp/pull/6350/commits/a88dbd0756f7f7e8db30a56ce22b215091684790